ui/components/multichain/import-tokens-popover/import-tokens-popover.js

```diff
@@ -0,0 +1,697 @@
+    setNftAddressError(null);
+    setSymbolAutoFilled(false);
+    setDecimalAutoFilled(false);
+    setMainnetTokenWarning(null);
+
+    const addressIsValid = isValidHexAddress(address, {
+      allowNonPrefixed: false,
+    });
+    const standardAddress = addHexPrefix(address).toLowerCase();
+
+    const isMainnetToken = Object.keys(STATIC_MAINNET_TOKEN_LIST).some(
+      (key) => key.toLowerCase() === address.toLowerCase(),
+    );
+
+    let standard;
+    if (addressIsValid) {
+      try {
+        ({ standard } = await getTokenStandardAndDetails(
+          standardAddress,
+          selectedAddress,
+          null,
+        ));
+      } catch (error) {
+        // ignore
+      }
+    }
+
+    const addressIsEmpty = address.length === 0 || address === EMPTY_ADDRESS;
+
+    switch (true) {
+      case !addressIsValid && !addressIsEmpty:
+        setCustomAddressError(t('invalidAddress'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case standard === TokenStandard.ERC1155 ||
+        standard === TokenStandard.ERC721:
+        setNftAddressError(
+          t('nftAddressError', [
+            <ButtonLink
+              className="import-tokens-popover__nft-address-error-link"
+              onClick={() =>
+                history.push({
+                  pathname: ADD_NFT_ROUTE,
+                  state: {
+                    addressEnteredOnImportTokensPage: customAddress,
+                  },
+                })
+              }
+              color={TextColor.primaryDefault}
+              key="nftAddressError"
+            >
+              {t('importNFTPage')}
+            </ButtonLink>,
+          ]),
+        );
+        break;
+
+      case isMainnetToken && !isMainnet:
+        setMainnetTokenWarning(t('mainnetToken'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case Boolean(identities[standardAddress]):
+        setCustomAddressError(t('personalAddressDetected'));
+        break;
+
+      case checkExistingAddresses(address, tokens):
+        setCustomAddressError(t('tokenAlreadyAdded'));
+        break;
+
+      default:
+        if (!addressIsEmpty) {
+          attemptToAutoFillTokenParams(address);
+          if (standard) {
+            setTokenStandard(standard);
+          }
+        }
+    }
+  };
+
+  // Determines whether to show the Search/Import or Confirm action
+  const isConfirming = mode === 'confirm';
+
+  return (
+    <Popover
+      onBack={isConfirming ? () => setMode('') : null}
+      onClose={() => {
+        dispatch(clearPendingTokens());
+        onClose();
+      }}
+      centerTitle
+      title={t('importTokensCamelCase')}
+      className="import-tokens-popover"
+    >
+      {isConfirming ? (
+        <Box padding={[0, 6, 6, 6]}>
```

Comment:
Is this intended to be responsive padding? If it's each side it will need to be listed out as `paddingTop={4} paddingRight={6} paddingBottom={4} paddingLeft={4}`. Array props syntax were updated to be used for responsive props so you could change the padding at different viewports without need the mixins or media queries